src/index.html

```diff
@@ -1,13 +1,431 @@
+                </h3>
+
+                <div class="testimonial__role">
+                  Chief Design Officer
+                </div>
+
+                <p class="testimonial about">
+                  When my lovely valley of steam rises and half-day sun is above an impermeable
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+      </section>
+
+      <section class="page__section contact-us" id="contact-us">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+
+          <div class="grid grid--desktop">
+            <div class="grid__item grid__item--1-2">
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55" class="contact-us__info-link"
+                >
+                  +1 234 5555-55-55
+                </a>
+              </div>
+              <div class="contact-us__info">
+                <div class="contact-us__info-title">
+                  Email
+                </div>
+                <a
+                  href="mail:hello@miami.com" class="contact-us__info-link"
+                >
+                  hello@miami.com
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--4-6">
+              <div class="contact-us__info contact-us__info--last">
+                <div class="contact-us__info-title">
+                  Address
+                </div>
+                <a
+                  href="https://goo.gl/maps/Nm2rZDFR2HEDPQiV9" class="contact-us__info-link"
+                >
+                  400 first ave. n.
+                  <br>
+                  suite 700
+                  <br>
+                  Minneapolis, MN 55401
+                </a>
+              </div>
+            </div>
+
+            <div class="grid__item grid__item--8-12">
+              <form action="">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contact-us__field form-field"
+                >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contact-us__field form-field"
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  class="contact-us__field contact-us__field--last form-field form-field--text"
+               ></textarea>
+
+                <button class="button button--primary">
+                  Consultation
+                </button>
+              </form>
+            </div>
+          </div>
+        </div>
+      </section>
+    </main>
+
+    <footer class="footer">
+      <a href="/" class="logo">
```

Comment:
```suggestion
      <a href="#" class="logo">
```
зараз при кліку видає 404 помилку, якщо поставити решітку то сторінка просто оновлюватиметься